css/styles.css

```diff
@@ -72,9 +85,269 @@ ul {
 
 .icons {
   display: flex;
-  align-items: center;
 }
 
 .icons li {
   padding-right: 12px;
 }
+
+/* Second page */
+
+.workTitle {
+  font-family: Crete-Round, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  left: 0;
+  top: 682px;
+  padding-left: 20px;
+}
+
+.yoga {
+  margin-top: 30px;
+  padding-left: 15px;
+}
+
+.border {
+  width: 327px;
+  height: 0;
+  left: 0;
+  top: 107px;
+  border: 1px solid #212121;
+  padding-left: 20px;
+}
+
+.overflow {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  padding-left: 20px;
+}
+
+.secondPage {
+  background: #e5e5e5;
+  display: grid;
+  grid-template-columns: 1fr;
+  width: 375px;
+}
+
+.secondItem {
+  background-image: url('secondBG.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-top: 40px;
+  margin-left: 4px;
+  padding: 20px;
+  width: 327px;
+}
```

Comment:
- [ ] [OPTIONAL] please try to stick to Figma design as much as possible, so you should make your button like the one in the Figma design 

- [ ] Your design 
![image](https://user-images.githubusercontent.com/61790587/164803540-b0735064-6ffd-4fff-919f-e6ae3b3d38af.png)

- [ ] Figma design 
![image](https://user-images.githubusercontent.com/61790587/164803558-ce0d84b2-5df9-4bd6-9c5b-fbd25c9ea0c7.png)